e2e-tests/tests/scheduled_backups/01-pxc.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 10
+commands:
+  - script: WATCH_NAMESPACE=$NAMESPACE ../../../bin/manager -metrics-bind-address :8087 -health-probe-bind-address :8088
+    background: true
+---
+apiVersion: dbaas.percona.com/v1
+kind: DatabaseCluster
+metadata:
+  name: test-pxc-cluster
+spec:
+  databaseType: pxc
+  databaseImage: percona/percona-xtradb-cluster:8.0.23-14.1
+  databaseConfig: |
+    [mysqld]
+    wsrep_provider_options="debug=1;gcache.size=1G"
+  secretsName: pxc-sample-secrets
+  clusterSize: 1
+  loadBalancer:
+    type: haproxy
+    exposeType: ClusterIP
+    size: 1
+    image: percona/percona-xtradb-cluster-operator:1.11.0-haproxy
+  dbInstance:
+    cpu: "1"
+    memory: 1G
+    diskSize: 15G
+  backup:
+    enabled: true
+    image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+    storages:
+      aws:
+        type: s3
+        storageProvider:
+          bucket: test
+          credentialsSecret: test-pxc-cluster-backup-s3
+          region: us-west-1
+      azure:
+        type: azure
+        storageProvider:
+          containerName: test
+          credentialsSecret: test-pxc-cluster-backup-azure
+          endpointUrl: https:/example.com
```

Comment:
Sorry for being pedantic but seeing a single `/` is triggering my OCD :sweat: (also in other test files)

Reply:
xD Good catch. Thanks!